source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,243 @@
+using the following snippet: 
+
+.. code-block:: java
+
+    collection.createIndex(Indexes.compoundIndex(Indexes.text("name"), Indexes.text("major")));
+
+.. note::
+
+   The student cannot create a second index because there can only be one
+   text index per collection. 
+
+For more information, see the following documentation:
+
+- :manual:`Text Indexes </core/index-text/>` Server Manual Entry
+- :ref:`Text Indexes <index-text-indexes>` in our Indexes builder guide
+- :ref:`Compound Indexes <index-compound-indexes>` in our Indexes builder guide
+
+Searching
+---------
+
+Use the ``Filters.text()`` method to perform a text search. 
+
+The ``Filters.text()`` method uses the :doc:`Filters builder
+</fundamentals/builders/filters>` to perform the text search on the
+fields indexed with the text index on the collection. The text search
+matches documents which contain one or more search terms.
+
+You can include ``TextSearchOptions`` as the second parameter to specify
+text search options such as case sensitivity. By default, there is no
+case sensitivity, meaning MongoDB matches documents regardless if the
+value is lowercase or uppercase. 
+
+For more information, see the following API documentation:
+
+- :java-core-api:`Filters.text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text By a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to search for
+a term in your text search. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. The ``Filters.text()`` method performs a logical OR on
+each term and returns documents that contain any of the terms. 
+
+Example 
+```````
+
+The student wants to find Science courses with case sensitivity on
+the first letter being capital.
+
+To address this scenario, the student performs the following:
+
+- Specify to have the search be case sensitive though ``TextSearchOptions``
+- Perform a text search on documents that contain the term "Science" with the ``Filters.text()`` method
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Mathematical Structures for Computer Science", "major": ["Computer Science", "Mathematics"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+
+.. note::
+
+   The output would consist of all the documents in the
+   ``junior_courses`` collection if the search was not case sensitive
+   because each document contains the term "Science" in the ``name``
+   field. 
```

Comment:
The order of information seems reversed. How about ordering the cause before the effect, e.g.:
"If the search was not case sensitive... the output would..."